src/gui/components/ChannelView.tsx

```diff
@@ -259,7 +259,28 @@ export default memo(function ChannelView({
             text: (
                 <span className={termstyles.promptTextColor}>
                     <br />
-                    Wait for peers to connect then type &quot;go&quot; to start:
+                    Wait for peers to connect then type &quot;go&quot; to start:{' '}
+                    <span
+                        className={termstyles.link}
+                        onClick={() => {
+                            new Promise<void>((resolve, reject) => {
```

Comment:
...or this Promise?

Reply:
fixed: https://github.com/playmint/playerchain-demo/pull/228/commits/1a29dc97b27938204c344e447dedad42e47231da#diff-7e1a03df4f03028f31d9e012d0db6d6674b94c94c0fa3160e4c020f4349ee19dL265-R272